.github/workflows/publish-charts.yaml

```diff
@@ -8,7 +8,7 @@ on:
 # Prevent two release workflows from running concurrently.
 concurrency: 
   group: chart-releaser
```

Comment:
Heavily out of scope, but I'd take advantage of the fact that no one is using this yet and release this as an OCI image instead https://github.com/roobre/kubeklipper/blob/main/.github/workflows/publish.yaml. Getting rid of chart-releaser and versions-as-text is like a breath of fresh air.

Reply:
The release process will be done `on: release`, creating a release either manually or automated using release-toolkit and a cronjob.

This is out of scope.